src/core/Akka.Remote.TestKit/DataTypes.cs

```diff
@@ -277,35 +270,37 @@ public override int GetHashCode()
 
         public Hello(string name, Address address)
         {
-            _name = name;
-            _address = address;
+            Name = name;
+            Address = address;
         }
 
-        public string Name
-        {
-            get { return _name; }
-        }
+        public string Name { get; }
+
+        public Address Address { get; }
 
-        public Address Address
+        public override string ToString()
         {
-            get { return _address; }
+            return $"Hello(Name: {Name}, Address: {Address})";
```

Comment:
Made this type pretty print-able. Honestly, could probably have just turned this into a `record` and saved myself the trouble.